### PR TITLE
fix error message

### PR DESCRIPTION
**Pull request by kirillseva**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: s3mpi
 Title: R message passing interface using S3 storage
 Description: Easily pass objects like lists or dataframes between consoles
-Version: 0.2.2
+Version: 0.2.3
```
Comment (peterhurford) [inline DESCRIPTION]: Maybe 0.2.2.1?  Don't know how @robertzk wants to do versioning.  Also don't know if he wants bugfixes in `NEWS.md`.

**Comment by kirillseva** [comment]
Definitely not 0.2.2.1
http://semver.org/

File: DESCRIPTION (hunk 1 above)
Comment (robertzk) [inline DESCRIPTION]: Good questions. I agree that semver is the way to go to be consistent with the rest of the ecosystem. If the bug is not major I think Hadley does not put it in `NEWS.md` either.

Comment (kirillseva) [inline DESCRIPTION]: who consumes NEWS.md?

**Pull request merged**